shardy/dialect/sdy/ir/attrs.td

```diff
@@ -670,7 +675,7 @@ def Sdy_DimMapping : AttrDef<Sdy_Dialect, "DimMapping"> {
     i.e. the dimension isn't mapped to any factors.
    }];
   let parameters = (ins
-    OptionalArrayRefParameter<"int64_t">:$factor_indices
+    OptionalArrayRefParameter<"int64_t", "list of factor indices">:$factor_indices
```

Comment:
we didn't say "list of" in other places, lets be consistent.

Also, shall we maybe say - factor indices this dimension is mapped to